tools/python/boutiques/localExec.py

```diff
@@ -1225,7 +1225,12 @@ def _findDOI(self, userIn):
             # Most recent DOI in file if user is publisher
             # Include check to ensure descriptor is unmodified
             if self.desc_dict.get('doi') is not None:
-                doi = self.desc_dict.get('doi')
+                # Zenodo returns the full DOI, but for the purposes of
+                # Boutiques we just use the Zenodo-specific portion (as its the
+                # unique part). If the API updates on Zenodo to no longer
+                # provide the full DOI, this still works because it just grabs
+                # the last thing after the split.
+                doi = self.desc_dict.get('doi').split('/')[-1]
```

Comment:
This should be called `zenodo_id` but otherwise all good!

Reply:
Agreed :)

Reply:
Actually, I should fix this in the puller, which actually is using the DOIs, since that's the place it fails. Everywhere else, we should use full DOIs or Zenodo IDs equivalently, and handle it appropriately in the `Puller`. Thoughts?